tests/end_to_end/server.py

```diff
@@ -194,6 +194,7 @@ def getCmdLine(self):
 python server_dev.py
 --dont-use-reloader
 --disable-urllib-warnings
+-d
```

Comment:
best to use the non-shorthand version of the flag here, for clarity in reading this file